generators/app/prompts.js

```diff
@@ -115,6 +117,26 @@ module.exports = [
     default: false,
     when: props => props.ui !== 'ionic'
   },
+  {
```

Comment:
Since i18n cannot be completely disabled in the current code base, I think we can remove this prompt for now and only ask which language to support

Reply:
I have no idea about this. Currently this is just like a shortcut to choose English language only quickly. 
follow your decision.

Reply:
I think the best compromise then is to remove this prompt and make english the (only) default selected language, so we can have the same behavior, 1 less prompt and no misleading as i18n cannot be really disabled :)

Reply:
Clear clever.